client/scss/components/_panel.scss

```diff
@@ -160,3 +160,21 @@ $header-button-size: theme('spacing.6');
 .w-panel__wrapper {
   @include max-form-width();
 }
+
+.w-panel--dashboard {
```

Comment:
What do you think of using a modifier that’s more generic like `w-panel--listing`?

Reply:
I'd leave it as it is for now to differentiate it from the old panel listings.